File: features/class_declaration_B/person_class.cpp
class Person {
    // Access specifier
    public:
 
    // Data Members
    string name;
 
    string greet(string other) { return other; }
	
    string present() {
       return "Yes";
    }
};
 
int main() { 
    // Declare an object of class geeks
    Person b;
	Person a;
	 
    // accessing data member
    b.name = "B";
	a.name = "A";
 		
    return 0;
}

File: features/class_declaration_B/person_class.swift
class Person {
    var name : String
    
	init(name: String) {
       self.name = name
    }
    
	func greet(other: String) -> String { return other }
	
	func present() -> String {
        return "Yes"
    }
	
}

val B = Person("B");
val A = Person("A");



